lerna.json

```diff
@@ -5,6 +5,7 @@
     "hero/client",
     "hero/core",
     "hero/interfaces",
+    "here/plugin-utils",
```

Comment:
I believe this should be "hero" not "here".

Reply:
Doh